internal/provider/none/installConfig.go

```diff
@@ -5,10 +5,36 @@ import (
 	"github.com/openshift/assisted-service/internal/common"
 	"github.com/openshift/assisted-service/internal/host/hostutil"
 	"github.com/openshift/assisted-service/internal/installcfg"
+	"github.com/openshift/assisted-service/internal/network"
 	"github.com/openshift/assisted-service/internal/provider"
 	"github.com/openshift/assisted-service/models"
+	"github.com/samber/lo"
 )
 
+func (p noneProvider) replaceMachineNetworkIfNeeded(cluster *common.Cluster, cfg *installcfg.InstallerConfigBaremetal) {
+	bootstrapHost := common.GetBootstrapHost(cluster)
+	if bootstrapHost != nil {
+		nodeIpAllocations, err := network.GenerateNonePlatformAddressAllocation(cluster, p.Log)
+		if err != nil {
+			p.Log.WithError(err).Warnf("failed to get node ip allocations for cluster %s", *cluster.ID)
+		} else {
+			allocation, ok := nodeIpAllocations[lo.FromPtr(bootstrapHost.ID)]
+			if ok {
```

Comment:
same as with else, just return in case it is not ok

Reply:
I think we should a pre-install validation.
If we add a validation for it, then if we reached here, it means the validation was ignored.  So I don't want to stop installation in this case.

Reply:
Not sure i understand, there is not code running if !ok so why not to change the code 
if !ok {
  log.
  return
}
and run code only in case allocation exists?

Reply:
Ahh sorry. I will change.